src/utils/base64.ts

```diff
@@ -1,10 +1,11 @@
-const chars = '0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ$_';
+const chars = '0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ_$';
 
-export function toBase64(num: number) {
+export function toBase64(num: number, base63 = false) {
```

Comment:
Is there any reason why we should not always use base63? As far as I can see, this is only used for deshadowing anyway so there should not be any issues. But using a function `toBase64` to convert to base 63 but only in some situations seems a little weird.